Dapper/DynamicParameters.cs

```diff
@@ -463,11 +463,13 @@ public DynamicParameters Output<T>(T target, Expression<Func<T, object>> express
 
                 if (parameters.TryGetValue(dynamicParamName, out parameter))
                 {
-                    parameter.ParameterDirection = parameter.AttachedParam.Direction = ParameterDirection.InputOutput;
+                    parameter.ParameterDirection = ParameterDirection.InputOutput;
+                    parameter.Size = sizeToSet;

```

Comment:
This is a behavior change, which may affect `max` column sizes. It's no longer conditional.